test/openAPI/features/support/wallet_generate_link_code.js

```diff
@@ -86,13 +98,14 @@ Then(
 // Scenario: Not able to generate link code because of a random transaction_id
 // Given and others Then for this scenario are written in the aforementioned example
 When(
-  'Send POST \\/linked-authorization\\/link-code request with given X-XSRF-TOKEN header, random transactionId and {string} as requestTime',
-  (requestTime) =>
+  'Send POST \\/linked-authorization\\/link-code request with given X-XSRF-TOKEN header, random transactionId and requestTime',
+  () =>
     specWalletGenerateLinkCode
       .post(baseUrl)
       .withHeaders(X_XSRF_TOKEN.key, X_XSRF_TOKEN.value)
+      .withCookies(X_XSRF_TOKEN.key, X_XSRF_TOKEN.value)
```

Comment:
cookies key should be 'XSRF-TOKEN' and request header should have 'X-XSRF-TOKEN', there is a small difference in the key. (Comment moved from old PR https://github.com/GovStackWorkingGroup/bb-identity/pull/31#discussion_r1241361022)